### ISSUE TITLE
About the PSNR and L1 loss value

### ISSUE THREAD

**Issue by momoda-mht** [thstkdgus35/EDSR-PyTorch#224]
Hi, when I run some models in the code, including but not limited to RCAN, the PSNR value seems to be lower and lower after 25 epochs, and the L1 loss vary a lot between different epochs as the below picture shown, is it normal?
![PSNR](https://user-images.githubusercontent.com/24655476/67160261-90f7cb00-f381-11e9-9483-48f6e175a573.png)

**Comment by AbdulMoqeet** [comment]
Set5 contains only 5 images, but in your case, it shows 40 images. Please make sure, you have correctly, set-up the training and test data.

**Comment by momoda-mht** [comment]
thank you for your reply. My Set5 contain 40 images because I add some other images, and the cause for getting lower PSNR value may be the way I add to the Set5 is not correct. But I still have the problem that the L1 loss vary a lot between epochs. By the way, when I use the model 'rcan.py' in the 'model' profile for training, after several epochs, the L1 loss suddenly increase, and the PSNR value drop a lot, is there any bug in data loading?
![QQ图片20200106162206](https://user-images.githubusercontent.com/24655476/71805339-f4e88080-30a0-11ea-900b-d5e95f36392a.png)

**Comment by LY54** [comment]
@momoda-mht Hello, I have the same problem. Have you solved it? What's your suggestion

**Comment by 1187697147** [comment]
I guess you use RCAN pre-trained model? It seems to use lower learning rate to load pre-trained RCAN.pt model.
